### ISSUE TITLE
Travis-CI: OSX build failure at autogen.sh

### ISSUE THREAD

**Issue by shantonu** [ofiwg/libfabric#2359]
I am seeing the following failure in OSX which is unrelated to the PR #2358. Not sure if it because of any change in Travis's OSX environment. Any idea?

```
$ ./autogen.sh
+autoreconf -ivf
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I config
autoreconf: configure.ac: tracing
autoreconf: running: glibtoolize --copy --force
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 1419: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 2247: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 401: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 407: /usr/local/Library/ENV/4.3/sed: No such file or directory
/usr/local/bin/glibtoolize: line 2023: /usr/local/Library/ENV/4.3/sed: No such file or directory
autoreconf: running: /usr/local/Cellar/autoconf/2.69/bin/autoconf --force
autoreconf: running: /usr/local/Cellar/autoconf/2.69/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:60: installing 'config/ar-lib'
configure.ac:60: installing 'config/compile'
configure.ac:15: installing 'config/config.guess'
configure.ac:15: installing 'config/config.sub'
configure.ac:12: installing 'config/install-sh'
configure.ac:92: error: required file 'config/ltmain.sh' not found
configure.ac:12: installing 'config/missing'
Makefile.am: installing 'config/depcomp'
parallel-tests: installing 'config/test-driver'
autoreconf: automake failed with exit status: 1
The command "./autogen.sh" failed and exited with 1 during .
```

**Comment by jsquyres** [comment]
This might be a problem at Travis and/or in our `.travis.yml` file.  See #2360.

**Comment by shefty** [comment]
Travis is still reporting failures for OS X.  I've tried re-starting Travis on the opened PR, with the results being the same.  Travis is reporting delays processing OS X, but no other status.

**Comment by shantonu** [comment]
I think the PR needs to be rebased with master. If it doesn't have Jeff's patch, it will fail.

**Comment by jsquyres** [comment]
@shefty Are you seeing this on *new* PRs (that we created after I submitted the Travis fix last night)?  Or only on PRs that were created before the Travis fix from last night?

...heh, @shantonu just beat me to the comment.  😄 

FWIW, however, I *may* have seen behavior before that implied that Travis may be caching the .travis.yml file *when the PR is created*, such that even if you rebase and get a new .travis.yml file in the base of your PR, Travis might not see or use it.

Meaning:

1. If you're only seeing this behavior on *old* PRs
1. If you rebase the old PRs and you *still* see this behavior
1. *New* PRs do not show this behavior

then it's time for 🍺.
